src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/GameTuningConfig.java

```diff
@@ -93,6 +99,17 @@ private void wireupChunkLimits(ConfigurationSection config) {
 		}
 	}
 
+	/**
+	 * Wireup for enabling setting your spawn during the day.
+	 *
+	 * @author Amelorate
+     */
+	private void wireupDaytimeBed(ConfigurationSection config) {

```

Comment:
You should do a null check here in case the config doesnt specify a daytimebed section at all